backends/arm/test/ops/test_add.py

```diff
@@ -142,7 +149,9 @@ def test_add2_tosa_BI(self, operand1: torch.Tensor, operand2: torch.Tensor):
         test_data = (operand1, operand2)
         self._test_add_tosa_BI_pipeline(self.Add2(), test_data)
 
+    # TODO(MLETORCH-352) Remove @unittest.expectedFailure when bug fixed in Regor
```

Comment:
Can these tickets be mirrored on ET GH Issues? Or just description of the bug and corresponding MLE ticket in GH Issue?

Reply:
This shouldn't be needed anymore - as some of these TODOs have been removed. I have amended one of the TODOs in lieu of a GitHub issue